packages/user/src/components/UsersTable/AllUsersTable.tsx

```diff
@@ -162,13 +162,29 @@ export const AllUsersTable = ({
       accessorKey: "status",
       header: t("table.defaultColumns.status"),
       cell: ({ row: { original } }) => {
+        const getValue = () => {
+          if (original.isActiveUser) {
+            if (original.disabled) {
+              return t("status.disable");
+            } else {
+              return t("status.active");
+            }
+          } else {
+            return t("status.invited");
+          }
+        };
+
+        const severity = original.isActiveUser
+          ? original.disabled
+            ? "danger"
+            : "success"
+          : undefined;
+
```

Comment:
using nested ternary operator is not recommend. Lets add a method for this similar to getValue above.